pkg/schemas/model.go

```diff
@@ -91,11 +91,11 @@ type Type struct {
 	Version string `json:"$schema,omitempty"` // section 6.1
 	Ref     string `json:"$ref,omitempty"`    // section 7
 	// RFC draft-wright-json-schema-validation-00, section 5
-	MultipleOf           int              `json:"multipleOf,omitempty"`           // section 5.1
-	Maximum              float64          `json:"maximum,omitempty"`              // section 5.2
-	ExclusiveMaximum     bool             `json:"exclusiveMaximum,omitempty"`     // section 5.3
-	Minimum              float64          `json:"minimum,omitempty"`              // section 5.4
-	ExclusiveMinimum     bool             `json:"exclusiveMinimum,omitempty"`     // section 5.5
+	MultipleOf           *float64         `json:"multipleOf,omitempty"`           // section 5.1
+	Maximum              *float64         `json:"maximum,omitempty"`              // section 5.2
+	ExclusiveMaximum     *float64         `json:"exclusiveMaximum,omitempty"`     // section 5.3
```

Comment:
Changing Exclusive* to floats here means dropping support for draft 4 in this respect. No objections as long as it's a conscious choice. Maybe someone can figure out a clean (enough) way to support both 4 and newer? Union types would help.